client-src/elements/chromedash-feature-filter.js

```diff
@@ -252,16 +238,16 @@ class ChromedashFeatureFilter extends LitElement {
         margin: 6px 3px;
       }
       .cond-field-menu {
-        width: 160px;
+        width: 180px;
       }
       .cond-op-menu {
-        width: 80px;
+        width: 150px;
       }
       .filterrow label {
         padding-right: 4px;
         align-self: center;
       }
-      .filterrow input {
+      .filterrow sl-input {
```

Comment:
merge the `sl-input` to the `sl-input` in line 222

Reply:
They are very different CSS selectors.  The one on line 222 is for icons inside the search query input field, whereas this one is for query values.  CSS can be so confusing sometimes.  I am glad that Lit allows us to deal with small pieces of it.